### PR TITLE
add new terms

### LINKED ISSUE #7195: Request for new term [Hemiplegic migraine-developmental and epileptic encephalopathy spectrum]
### Label

hemiplegic migraine-developmental and epileptic encephalopathy spectrum

### Synonyms

_No response_

### Synonym type

None

### Definition

A spectrum in which individuals may present with phenotypes ranging from hemiplegic migraines without epilepsy to developmental and epileptic encephalopathy with or without episodic hemiplegia or other forms of paresis. Symptoms and severity may vary within families.

### Parent term

MONDO:0100062 developmental and epileptic encephalopathy, MONDO:0018925 familial or sporadic hemiplegic migraine

### Children term(s)

_No response_

### ORCID Identifier

_No response_

### Website URL

https://clinicalgenome.org/affiliation/40005/

### Additional comments

We are hoping to have two parent terms (listed in the Parent term field). This request is being submitted on behalf of the ClinGen Epilepsy Gene Curation Expert Panel, and is intended to capture the phenotypic spectrum associated with genes like _ATP1A2_ (PMIDs: 24096472, 18056581, 33880529).

CC: @ErinRiggs

### LINKED ISSUE #7218: GATA6- related congenital heart disease with or without pancreatic agenesis or neonatal diabetes
GATA6- related congenital heart disease with or without pancreatic agenesis or neonatal diabetes

**Parent term (use [OLS](https://www.ebi.ac.uk/ols/ontologies/mondo), or your favorite ontology browser)**
congenital heart disease 

**Definition**
A heart disease that is present at birth. Representative examples include atrial septal defect 9, conotruncal heart malformations, tetralogy of Fallot, ventricular septal defect, atrioventricular septal defect, bicuspid aortic valve, transposition of the great arteries, persistent truncus arteriosus, congenital heart disease with pancreatic agenesis, and congenital heart disease with neonatal diabetes

**Definition source (Please give PubMed ID, if applicable, in format PMID:#######)
PMIDs: 19666519, 23020118, 25937001, 28049534, 29653232, 31301121, 35595280

**Your nano-attribution (ORCID) or URL for a working group**
https://clinicalgenome.org/affiliation/40130/

### LINKED ISSUE #7242: GATA5 related congenital heart defects/ GATA5
**Preferred gene-related syndrome label**
GATA5 related congenital heart defects

**Synonyms**

**Parent term (use [OLS](https://www.ebi.ac.uk/ols/ontologies/mondo), or your favorite ontology browser)**
MONDO:0005453

**Definition**
Please write the definition in the format: Any [parent class] in which the cause of the disease is a mutation in the [gene name] gene. For example: Any congenital myopathy in which the cause of the disease is a mutation in the TPM3 gene.

A heart disease that is present at birth. Representative examples include tetralogy of fallot, bicuspid aortic valve, atrial septal defect, double outlet right ventricle, ventricular septal defect, and coarctation of the aorta, and atrioventricular canal. 

**Definition source (Please give PubMed ID, if applicable, in format PMID:#######)
PMIDs: 22641149, 24638895, 23031282, 22961344, 23289003

**Children terms (if applicable)** Should any existing terms be re-classified as children underneath this new proposed term? 

**Your nano-attribution (ORCID) or URL for a working group**
https://clinicalgenome.org/affiliation/40130/

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
close #7195
close #7218
close #7242
close https://github.com/monarch-initiative/mondo/issues/7137

[ROBOT spreadsheet](https://docs.google.com/spreadsheets/d/1PzCakL380vKgPmiKx78X0OR_3yszzuS7ytHztiIvZVc/edit#gid=1650269388)

recreates PR: https://github.com/monarch-initiative/mondo/pull/7194

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -513981,6 +513981,57 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
...
+name: GATA5-related congenital heart defects
+def: "A congenital heart disease that is present at birth. Representative examples include tetralogy of fallot, bicuspid aortic valve, atrial septal defect, double outlet right ventricle, ventricular septal defect, and coarctation of the aorta, and atrioventricular canal." [https://clinicalgenome.org/affiliation/40130/, PMID:22641149, PMID:22961344, PMID:23031282, PMID:23289003, PMID:24638895]
+synonym: "GATA5 related congenital heart defects" EXACT [https://clinicalgenome.org/affiliation/40130/]
+is_a: MONDO:0005453 {source="https://clinicalgenome.org/affiliation/40130/"} ! congenital heart disease
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7242" xsd:anyURI
+
+[Term]
+id: MONDO:0100542
+name: clonal hematopoiesis
+def: "A precancerous condition characterized by cellular proliferation of hematopoietic cells where a substantial proportion of the cells is derived from a single hematopoietic stem cell lineage." [NCIT:C162188, PMID:34215849]
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:514012]: nit:  cells is derived -->  cells are derived

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -513981,6 +513981,57 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
...
+id: MONDO:0100539
+name: hemiplegic migraine-developmental and epileptic encephalopathy spectrum
+def: "A spectrum in which individuals may present with phenotypes ranging from hemiplegic migraines without epilepsy to developmental and epileptic encephalopathy with or without episodic hemiplegia or other forms of paresis. Symptoms and severity may vary within families." [https://clinicalgenome.org/affiliation/40005/]
+is_a: MONDO:0018925 {source="https://clinicalgenome.org/affiliation/40005/"} ! familial or sporadic hemiplegic migraine
+is_a: MONDO:0100062 {source="https://clinicalgenome.org/affiliation/40005/"} ! developmental and epileptic encephalopathy
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7195" xsd:anyURI
+
+[Term]
+id: MONDO:0100540
+name: GATA6-related congenital heart disease with or without pancreatic agenesis or neonatal diabetes
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:513994]: See comment in ticket: https://github.com/monarch-initiative/mondo/issues/7218#issuecomment-1933116976

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -513981,6 +513981,57 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
...
+def: "A congenital heart disease that is present at birth. Representative examples include atrial septal defect 9, conotruncal heart malformations, tetralogy of Fallot, ventricular septal defect, atrioventricular septal defect, bicuspid aortic valve, transposition of the great arteries, persistent truncus arteriosus, congenital heart disease with pancreatic agenesis, and congenital heart disease with neonatal diabetes." [https://clinicalgenome.org/affiliation/40130/, PMID:19666519, PMID:23020118, PMID:25937001, PMID:28049534, PMID:29653232, PMID:31301121, PMID:35595280]
+synonym: "GATA6 related congenital heart disease with or without pancreatic agenesis or neonatal diabetes" EXACT [https://clinicalgenome.org/affiliation/40130/]
+is_a: MONDO:0005453 {source="https://clinicalgenome.org/affiliation/40130/"} ! congenital heart disease
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4174 {source="https://clinicalgenome.org/affiliation/40130/"} ! GATA6
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7218" xsd:anyURI
+
+[Term]
+id: MONDO:0100541
+name: GATA5-related congenital heart defects
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:514003]: See comment in ticket: https://github.com/monarch-initiative/mondo/issues/7242#issuecomment-1933120978

File: src/ontology/mondo-edit.obo [hunk 4]
```diff
@@ -513981,6 +513981,57 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
...
+def: "A congenital heart disease that is present at birth. Representative examples include tetralogy of fallot, bicuspid aortic valve, atrial septal defect, double outlet right ventricle, ventricular septal defect, and coarctation of the aorta, and atrioventricular canal." [https://clinicalgenome.org/affiliation/40130/, PMID:22641149, PMID:22961344, PMID:23031282, PMID:23289003, PMID:24638895]
+synonym: "GATA5 related congenital heart defects" EXACT [https://clinicalgenome.org/affiliation/40130/]
+is_a: MONDO:0005453 {source="https://clinicalgenome.org/affiliation/40130/"} ! congenital heart disease
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7242" xsd:anyURI
+
+[Term]
+id: MONDO:0100542
+name: clonal hematopoiesis
```
Comment (twhetzel) [inline src/ontology/mondo-edit.obo:514011]: See comment in issue: https://github.com/monarch-initiative/mondo/issues/7137#issuecomment-1933129034

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:514012]: fixed

**Comment by nicolevasilevsky** [comment]
@twhetzel is further action needed on this on my end?

**Review by twhetzel — APPROVED** [review]
lgtm

**Pull request merged**